src/System.Diagnostics.EventLog/src/System/Diagnostics/EventLog.cs

```diff
@@ -667,6 +667,11 @@ public static EventLog[] GetEventLogs(string machineName)
                     handle.Close();
                     logs.Add(log);
                 }
+                else if (Marshal.GetLastWin32Error() != 87)
+                {
+                    // Windows returns ERROR_INVALID_PARAMETER for special keys which were added in RS5+ but do not represent actual event logs.
```

Comment:
The comment should explain why you are including logs you can't open (except when invalid parameter)